deploy/cloudformation/app-infrastructure.yml

```diff
@@ -110,6 +110,226 @@ Resources:
       RetentionInDays: 7
       LogGroupName: !Join [ '-', [ !Ref 'AWS::StackName', 'LogGroup']]
 
+  # Shared Roles/Groups
+  DataAdminRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Principal:
+            AWS:
+              - !Ref AWS::AccountId
+          Action:
+          - sts:AssumeRole
+      Path: '/'
+
+  DataAdminGroup:
+    Type: AWS::IAM::Group
+
+  DataAdminPermissions:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: A policy that allows the DataAdminRole to read all logs and other global actions
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: AllowGlobals # Things that don't support resource level permissions
+            Effect: Allow
+            Action:
+              - 's3:ListAllMyBuckets'
+            Resource: '*'
+      Roles:
+        - !Ref DataAdminRole
+
+  CanAssumeDataAdminRole:
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: A policy that allows a user or group to assume the DataAdminRole
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          -
+            Effect: Allow
+            # We may need to remove/rework this if we want to allow attaching this to a federated user's role
+            Condition:
+              BoolIfExists:
+                aws:MultiFactorAuthPresent: true
```

Comment:
This might be an organizational choice - MFA is best practice but do we want to dictate that orgs have to use it in order to adopt these role-based permission models?

Reply:
I think we should wait until we see if this is a problem. Err on the side of extra security, until we see a need. Unless you see a need for us to do this now?

Reply:
If the README gets updated to indicate that MFA is a requirement to assume these roles, then I'm fine with it.

Reply:
Sounds good, will add something